src/Microsoft.PowerShell.Commands.Utility/commands/utility/UnblockFile.cs

```diff
@@ -119,18 +120,13 @@ protected override void ProcessRecord()
                     {
                         AlternateDataStreamUtilities.DeleteFileStream(path, "Zone.Identifier");
                     }
-                    catch (Win32Exception accessException)
+                    catch (IOException)
```

Comment:
`File.Delete` ignores `File_Not_Found` error and throw exception for all other errors, see: http://source.dot.net/#System.IO.FileSystem/System/IO/FileSystem.Windows.cs,30dc9ae494d8f0a5

So, we should use `catch (Exception e)` here and write a non-terminating error. The error ID `RemoveItemUnauthorizedAccessError` and `ErrorCategory.PermissionDenied` may not be appropriate, we need a more general error.